packages/core/test/financial-templates/contract-for-difference/ContractForDifference.js

```diff
@@ -29,7 +29,7 @@ let collateralWhitelist;
 let identifierWhitelist;
 let optimisticOracle;
 let finder;
-let ancillaryData;
+let ancillaryData = web3.utf8ToHex("CUSTOM_DATA");
```

Comment:
nit: thoughts on starting to make test data resemble how we actually intend to use ancillary data in prod, such as {key:value, key2:value2, ...}

Reply:
+1 -- will fix this

Reply:
Done!